### PR TITLE
chore: transferring ownership of presences

### PR DESCRIPTION
**Pull request by fr0r**
**Describe the changes in this pull request:**
A couple of days ago, I was asked by DoomLorD if I wanted to take over his [Discord applications](https://i.imgur.com/rLgI7YP.png), to which I've asked if he also wanted me to take over some of his presences which were still 'abandoned', to which he replied 'yes'.

The only changes made were in Anghami's metadata, which has recently [rebranded](https://twitter.com/anghami/status/1499294602850934788) with a new logo and stuff, the rest should work as intended.

I've also decided to abandon/change ownership of some of my presences, to which I don't have any interest in updating them any time soon.

<details>
<summary> Proof showing the creation/modification is working as expected </summary>
<br>

image

image

image

image

image
</details>

### CODE REVIEW

**Comment by fr0r** [comment]
do i really need to bump them all?

File: websites/T/Tsuki Mangás/dist/metadata.json [hunk 1]
```diff
@@ -1,23 +1,17 @@
 {
 	"$schema": "https://schemas.premid.app/metadata/1.6",
 	"author": {
-		"name": "fror",
-		"id": "515668127829458945"
+		"name": "Slowftw",
```
Comment (EncryptedDev) [inline websites/T/Tsuki Mangás/dist/metadata.json:4]: Unless you have been given explicit approval from Slowftw, you cannot do this. Transfer the presence to the PreMiD bot account and leave Slowftw as a contributor

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/T/Tsuki Mangás/dist/metadata.json (hunk 1 above)
Comment (fr0r) [reply websites/T/Tsuki Mangás/dist/metadata.json:4]: I have talked with @Slowftw and he agreed to take over it.

Comment (Slowftw) [reply websites/T/Tsuki Mangás/dist/metadata.json:4]: yo I agree 🤙

**Review by EncryptedDev — APPROVED** [review]
Approved. Thank you for your contribution

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**